modules/core/02-client/client/cli/tx.go

```diff
@@ -461,3 +457,96 @@ func NewCmdSubmitUpgradeProposal() *cobra.Command {
 
 	return cmd
 }
+
+// newCmdScheduleIBCUpgradeProposal defines the command for submitting an IBC software upgrade proposal.
+func newCmdScheduleIBCUpgradeProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "schedule-ibc-upgrade [name] [height] [path/to/upgraded_client_state.json] [flags]",
+		Args:  cobra.ExactArgs(3),
+		Short: "Submit an IBC software upgrade proposal",
+		Long: "Please specify a unique name and height for the upgrade to take effect.\n" +
+			"The client state specified is the upgraded client state representing the upgraded chain\n" +
+			`Example Upgraded Client State JSON: 
+{
+"@type":"/ibc.lightclients.tendermint.v1.ClientState",
+ "chain_id":"testchain1",
+"unbonding_period":"1814400s",
+"latest_height":{"revision_number":"0","revision_height":"2"},
+"proof_specs":[{"leaf_spec":{"hash":"SHA256","prehash_key":"NO_HASH","prehash_value":"SHA256","length":"VAR_PROTO","prefix":"AA=="},"inner_spec":{"child_order":[0,1],"child_size":33,"min_prefix_length":4,"max_prefix_length":12,"empty_child":null,"hash":"SHA256"},"max_depth":0,"min_depth":0},{"leaf_spec":{"hash":"SHA256","prehash_key":"NO_HASH","prehash_value":"SHA256","length":"VAR_PROTO","prefix":"AA=="},"inner_spec":{"child_order":[0,1],"child_size":32,"min_prefix_length":1,"max_prefix_length":1,"empty_child":null,"hash":"SHA256"},"max_depth":0,"min_depth":0}],
+"upgrade_path":["upgrade","upgradedIBCState"],
+}
```

Comment:
I think we should format this to make it as easy to read as possible.

```suggestion
{
   "@type":"/ibc.lightclients.tendermint.v1.ClientState",
   "chain_id":"testchain1",
   "unbonding_period":"1814400s",
   "latest_height":{
      "revision_number":"0",
      "revision_height":"2"
   },
   "proof_specs":[
      {
         "leaf_spec":{
            "hash":"SHA256",
            "prehash_key":"NO_HASH",
            "prehash_value":"SHA256",
            "length":"VAR_PROTO",
            "prefix":"AA=="
         },
         "inner_spec":{
            "child_order":[
               0,
               1
            ],
            "child_size":33,
            "min_prefix_length":4,
            "max_prefix_length":12,
            "empty_child":null,
            "hash":"SHA256"
         },
         "max_depth":0,
         "min_depth":0
      },
      {
         "leaf_spec":{
            "hash":"SHA256",
            "prehash_key":"NO_HASH",
            "prehash_value":"SHA256",
            "length":"VAR_PROTO",
            "prefix":"AA=="
         },
         "inner_spec":{
            "child_order":[
               0,
               1
            ],
            "child_size":32,
            "min_prefix_length":1,
            "max_prefix_length":1,
            "empty_child":null,
            "hash":"SHA256"
         },
         "max_depth":0,
         "min_depth":0
      }
   ],
   "upgrade_path":[
      "upgrade",
      "upgradedIBCState"
   ]
}
```

Reply:
Did you get this output from running the cli? You could also just pipe the output to `jq` and paste that.

Reply:
great idea, will add!